marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -188,54 +199,40 @@ describe('json scenario for table on browse documents page', () => {
     })
   });
 
-  xit('select "all entities" and verify table', () => {
+  it('select "all entities" and verify table', () => {
     browsePage.getSelectedEntity().should('contain', 'All Entities');
-    cy.wait(2000);
     browsePage.getHubPropertiesExpanded();
     browsePage.getTotalDocuments().should('be.greaterThan', 25)
     //check table rows
     browsePage.getTableRows().should('have.length', 20);
     //check table columns
     browsePage.getTableColumns().should('have.length', 5);
-    //check cells data
-    for (let i = 2; i <= 10; i++) {
-      for (let j = 2; j <= 4; j++) {
-        browsePage.getTableCell(i, j).should('not.be.empty')
-      }
-    }
-    for (let i = 1; i <= 10; i++) {
-      browsePage.getTableUriCell(i).should('not.be.empty')
-    }
   });
 
-  xit('select Person entity and verify table', () => {
+  it('select Person entity and verify table', () => {
     browsePage.selectEntity('Person');
     browsePage.getSelectedEntity().should('contain', 'Person');
-    cy.wait(2000);
     browsePage.getHubPropertiesExpanded();
     browsePage.getTotalDocuments().should('be.greaterThan', 5)
     //check table rows
     browsePage.getTableRows().should('have.length', 14);
     //check table columns
-    browsePage.getTableColumns().should('have.length', 5);
-    //check cells data
-    for (let i = 1; i <= 5; i++) {
-      for (let j = 2; j <= 5; j++) {
-        browsePage.getTableCell(i, j).should('not.be.empty')
-      }
-    }
+    browsePage.getTableColumns().should('to.have.length.of.at.most', 6);
```

Comment:
Thanks for removing the checks for table cells in for loop since there is a valid scenario for cells to be empty.